app/components/card_component.rb

```diff
@@ -0,0 +1,19 @@
+class CardComponent < ApplicationComponent
+  def initialize(classes: nil)
+    @class_names = classes
+  end
+
+  private
+
+  attr_reader :class_names
+
+  def classes
+    classes = ["card"]
+    classes.push(class_names) unless class_names.try(:empty?)
```

Comment:
```suggestion
    classes.push(class_names) unless class_names.empty?
```

Reply:
I think you can call .empty? on nil right?

Reply:
You cannot :)

Reply:
But I can probably refactor this now that I look at it